xbmc/addons/kodi-addon-dev-kit/include/kodi/tools/Time.h

```diff
@@ -0,0 +1,90 @@
+#pragma once
+/*
+ *  Copyright (C) 2005-2019 Team Kodi
+ *  Copyright (C) 2011-2012 Pulse-Eight Limited.
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#if defined(TARGET_DARWIN)
+#include <mach/mach_time.h>
+#include <CoreVideo/CVHostTime.h>
+#elif defined(TARGET_WINDOWS)
+#include <Windows.h>
+#include <time.h>
+#else
+#include <time.h>
+#endif
+
+namespace kodi
+{
+  namespace time
```

Comment:
namespaces are not indented, https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#32-indentation